src/main/resources/plugin.properties

```
@@ -5,3 +5,4 @@ description=${project.description}
 class=io.gravitee.resource.cache.NodeCacheResource
 type=resource
 icon=resource-cache.svg
+category=cache
```

I don't know if the case is important but for Authentication you chose camel case
```suggestion
category=Cache
```